packages/components/src/FormatRelativeDateTime/FormatRelativeDateTime.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { CivilDateTime } from "@std-proposal/temporal";
+
+interface FormatRelativeDateTimeProps {
+  /**
+   * Date to be displayed.
+   */
+  readonly date: CivilDateTime;
+}
+
+export function FormatRelativeDateTime(
+  civilDateTime: FormatRelativeDateTimeProps,
+) {
+  const now = Date.now() / 1000; //seconds
+  const inputDate = new Date(civilDateTime.date.toJSON());
+  const delta = now - inputDate.getTime() / 1000; //seconds;
+
+  switch (relativeTimeRange(delta)) {
+    case "less than an hour":
+      return showMinutes(Math.round(delta / 60));
+    case "less then a day":
+      return <>{inputDate.toLocaleTimeString()}</>;
```

Comment:
because you are not returning any elements, you can remove all the fragments (`<></>`) and just do:

```
return inputDate.toLocaleTimeString()
```

Reply:
Same will go for all return statements

Reply:
If I drop the <> and </>, then I get compile errors. In particular, I could find no way to get the following to compile without the <> and </>
`<>{strFormatDate(inputDate, { month: "short", day: "numeric" })}</>`

Reply:
If you take out the `<>` and the `{}` and just return the function:

```
return strFormatDate(inputDate, { month: "short", day: "numeric" });
```

This should work. Would be happy to jump on a chat to get working as well.

Reply:
@eddysims Got it now, I was keeping the extra { } surrounding the whole thing.